private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -214,3 +221,19 @@ func inverseEncoding(encoding bufconvert.MessageEncoding) (bufconvert.MessageEnc
 		return 0, fmt.Errorf("unknown message encoding %v", encoding)
 	}
 }
+
+// wktToPath returns the import path of the proto file for a well known type.
+func wktToPath(fulltype string) string {
+	msgType, err := protoregistry.GlobalTypes.FindMessageByName(protoreflect.FullName(fulltype))
+	if err != nil {
+		// not a statically known type
+		return ""
+	}
+	filePath := msgType.Descriptor().ParentFile().Path()
+	if datawkt.Exists(filePath) {
+		// found it!
+		return filePath
+	}
+	// not a well-known type
+	return ""
```

Comment:
```suggestion
	if !datawkt.Exists(filePath) {
		// not a well-known type
		return ""
	}
	// found it!
	return filePath
```

left align happy path